docs_manual/source/metadata_advanced.rst

```diff
@@ -107,6 +108,7 @@ SODAR currently supports the following assay plugins:
 - **Microarray**
 - **Protein Expression Profiling / Mass Cytometry**
 - **Protein Expression Profiling / Mass Spectrometry**
+- **Universal Assay Plugin**
```

Comment:
Let's call this "Generic Assay Plugin" to stay consistent with SODAR language and the internal naming.

This might initially make it confusing with the "Generic Raw Data Plugin". However, I will handle it by renaming/removing that plugin in a subsequent commit. It isn't currently used in any project in our SODAR instance and we could simply override its functionality with this plugin anyway.